gabc/details.html

```diff
@@ -121,7 +121,7 @@ <h3 id="signs">Rhythmic signs</h3>
 <p>In some scores, there are sometimes two levels of horizontal episema: a short episema beneath a longer one. To put an horizontal episemus under a note, type <strong><code>&nbsp;_0&nbsp;</code></strong>. This can be combined with a normal horizontal episemus.</p>
```

Comment:
The period at the end of the previous line (line 119) can wrap to the next line because there is a space before it.